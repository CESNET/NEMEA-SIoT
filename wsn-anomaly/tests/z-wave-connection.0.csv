double ACK_COUNT,double ACK_WAITING,double CAN_COUNT,uint64 DEV_ADDR,double NAK_COUNT,double OOF_COUNT,double SOF_COUNT,time TIMESTAMP,double AVERAGERequestRTT,double AVERAGE_RESPONSE_RTT,double BAD_CHECKSUM,double BAD_ROUTES,double BROADCAST_READ_COUNT,double BROADCAST_WRITE_COUNT,double CALLBACKS,double DROPPED,double LAST_REQUEST_RTT,double LAST_RESPONSE_RTT,double NET_BUSY,double NO_ACK,double NON_DELIVERY,double NOT_IDLE,double QUALITY,double READ_ABORTS,double READ_COUNT,double RECEIVE_DUPLICATIONS,double RECEIVE_UNSOLICITED,double RECEIVED_COUNT,double RETRIES,double ROUTED_BUSY,double SENT_COUNT,double SENT_FAILED,double WRITE_COUNT
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:21.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:21.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:21.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:21.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:26.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:26.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:26.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:26.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:31.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:31.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:31.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:31.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:36.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:36.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:36.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:36.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:41.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:41.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:41.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:41.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:46.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:46.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:46.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:46.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:51.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:51.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:51.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:51.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:56.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:56.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:56.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:56.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:01.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:01.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:01.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:01.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:06.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:06.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:06.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:06.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:11.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:11.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:11.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:11.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:16.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,2.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:16.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:16.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:16.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:21.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,2.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:21.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:21.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,136.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:21.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:26.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,2.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:26.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:26.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,236.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
