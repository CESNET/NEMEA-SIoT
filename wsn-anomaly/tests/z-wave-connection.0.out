2.000000,1113,0.000000,2018-04-05T15:18:16.000,6,"The device 1113 exceeded the maximal hard limit by 1.000000","MOVING_MEDIAN","DROPPED"
96.000000,22,36.000000,2018-04-05T15:18:26.000,7,"The device 22 exceeded the grow up limit with value 2.666667","MOVING_AVERAGE","LAST_RESPONSE_RTT"
